[2025-09-18T06:37:08.129+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: batch_ingestion_dag.load_to_minio manual__2025-09-18T06:26:34.314177+00:00 [queued]>
[2025-09-18T06:37:08.136+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: batch_ingestion_dag.load_to_minio manual__2025-09-18T06:26:34.314177+00:00 [queued]>
[2025-09-18T06:37:08.136+0000] {taskinstance.py:2171} INFO - Starting attempt 2 of 3
[2025-09-18T06:37:08.145+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_to_minio> on 2025-09-18 06:26:34.314177+00:00
[2025-09-18T06:37:08.152+0000] {standard_task_runner.py:60} INFO - Started process 566 to run task
[2025-09-18T06:37:08.155+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'batch_ingestion_dag', 'load_to_minio', 'manual__2025-09-18T06:26:34.314177+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/batch_ingestion_dag.py', '--cfg-path', '/tmp/tmp3sglslsd']
[2025-09-18T06:37:08.157+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask load_to_minio
[2025-09-18T06:37:08.197+0000] {task_command.py:423} INFO - Running <TaskInstance: batch_ingestion_dag.load_to_minio manual__2025-09-18T06:26:34.314177+00:00 [running]> on host 3aa26d851e1b
[2025-09-18T06:37:08.257+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='batch_ingestion_dag' AIRFLOW_CTX_TASK_ID='load_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2025-09-18T06:26:34.314177+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-18T06:26:34.314177+00:00'
[2025-09-18T06:37:08.258+0000] {batch_ingestion_dag.py:133} INFO - Uploading CSV to MinIO (raw-data bucket)...
[2025-09-18T06:37:08.323+0000] {batch_ingestion_dag.py:147} INFO - Bucket creation skipped (possibly exists): An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it.
[2025-09-18T06:37:08.333+0000] {batch_ingestion_dag.py:150} INFO - File successfully uploaded to MinIO.
[2025-09-18T06:37:08.334+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-18T06:37:08.342+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=batch_ingestion_dag, task_id=load_to_minio, execution_date=20250918T062634, start_date=20250918T063708, end_date=20250918T063708
[2025-09-18T06:37:08.367+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-18T06:37:08.388+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
